docs/interpreter/spark.md

```diff
@@ -81,30 +76,24 @@ for example,
 
 
 
-<br />
-That's it. Zeppelin will work with any version of Spark and any deployment type without rebuild Zeppelin in this way. (Zeppelin 0.5.5-incubating release works up to Spark 1.5.1)
-
-Note that without exporting SPARK_HOME, it's running in local mode with included version of Spark. The included version may vary depending on the build profile.
+That's it. Zeppelin will work with any version of Spark and any deployment type without rebuilding Zeppelin in this way. ( Zeppelin 0.5.5-incubating release works up to Spark 1.5.1 )
```

Comment:
think we've since moved to support 1.5.2:
https://github.com/apache/incubator-zeppelin/blob/master/spark/pom.xml#L87

it's pretty error prone to document versions like this..